docsets/NetworkX/docset.json

```diff
@@ -1,15 +1,21 @@
 {
     "name": "NetworkX",
-    "version": "1.11",
-    "archive": "NetworkX.tgz",
+    "version": "2.1",
+    "archive": "versions/2.1/NetworkX.tgz",
     "author": {
-        "name": "Aziz Alto",
-        "link": "https://github.com/iamaziz"
+        "name": "AllanLRH",
+        "link": "https://github.com/AllanLRH"
     },
-    "aliases": [
-        "Python",
-        "Complex Netowrks",
-        "Dynamics",
-        "Structure",
-        ]
+    "aliases": [],
+
+    "specific_versions": [
+        {
+            "version": "2.1",
+            "archive": "versions/2.1/NetworkX.tgz",
+        },
+        {
+            "version": "1.11",
+            "archive": "versions/1.11/NetworkX.tgz",
```

Comment:
You need to add the actual `tgz` at this path. Moving the `txt` file around is not supported.

Reply:
How do I get that file?

Reply:
https://kapeli.com/feeds/zzz/user_contributed/build/NetworkX/NetworkX.tgz